packages/merge-styles/api/merge-styles.api.ts

```diff
@@ -0,0 +1,366 @@
+  textDecorationSkip?: ICSSRule | string;
+  textDecorationStyle?: ICSSRule | string;
+  // (undocumented)
+  textDecorationUnderline?: ICSSRule | string;
+  textEmphasis?: ICSSRule | string;
+  textEmphasisColor?: ICSSRule | string;
+  textEmphasisStyle?: ICSSRule | string;
+  textHeight?: ICSSRule | string;
+  textIndent?: ICSSRule | string;
+  // (undocumented)
+  textJustifyTrim?: ICSSRule | string;
+  // (undocumented)
+  textKashidaSpace?: ICSSRule | string;
+  textLineThrough?: ICSSRule | string;
+  textLineThroughColor?: ICSSRule | string;
+  textLineThroughMode?: ICSSRule | string;
+  textLineThroughStyle?: ICSSRule | string;
+  textLineThroughWidth?: ICSSRule | string;
+  textOverflow?: ICSSRule | string;
+  textOverline?: ICSSRule | string;
+  textOverlineColor?: ICSSRule | string;
+  textOverlineMode?: ICSSRule | string;
+  textOverlineStyle?: ICSSRule | string;
+  textOverlineWidth?: ICSSRule | string;
+  textRendering?: ICSSRule | string;
+  textScript?: ICSSRule | string;
+  textShadow?: ICSSRule | string;
+  textTransform?: ICSSRule | string;
+  textUnderlinePosition?: ICSSRule | string;
+  textUnderlineStyle?: ICSSRule | string;
+  top?: ICSSRule | number | string;
+  touchAction?: ICSSRule | string;
+  transform?: ICSSRule | string;
+  transformOrigin?: ICSSRule | string;
+  transformOriginZ?: ICSSRule | string;
+  transformStyle?: ICSSRule | string;
+  transition?: ICSSRule | string;
+  transitionDelay?: ICSSRule | string;
+  transitionDuration?: ICSSRule | string;
+  transitionProperty?: ICSSRule | string;
+  transitionTimingFunction?: ICSSRule | string;
+  unicodeBidi?: ICSSRule | string;
+  userFocus?: ICSSRule | string;
+  userInput?: ICSSRule | string;
+  userSelect?: ICSSRule | 'none' | 'auto' | 'text' | 'all' | 'contain';
+  verticalAlign?: ICSSRule | string;
+  visibility?: ICSSRule | string;
+  voiceBalance?: ICSSRule | string;
+  voiceDuration?: ICSSRule | string;
+  voiceFamily?: ICSSRule | string;
+  voicePitch?: ICSSRule | string;
+  voiceRange?: ICSSRule | string;
+  voiceRate?: ICSSRule | string;
+  voiceStress?: ICSSRule | string;
+  voiceVolume?: ICSSRule | string;
+  WebkitFontSmoothing?: 'none' | 'antialiased' | 'grayscale' | 'subpixel-antialiased';
+  whiteSpace?: ICSSRule | string;
+  whiteSpaceTreatment?: ICSSRule | string;
+  widows?: ICSSRule | number;
+  width?: ICSSRule | string;
+  wordBreak?: ICSSRule | string;
+  wordSpacing?: ICSSRule | string;
+  wordWrap?: ICSSRule | string;
+  wrapFlow?: ICSSRule | string;
+  wrapMargin?: ICSSRule | string;
+  wrapOption?: ICSSRule | string;
+  writingMode?: ICSSRule | string;
+  zIndex?: ICSSRule | 'auto' | number;
+  zoom?: ICSSRule | 'auto' | number | ICSSPercentageRule;
+}
+
+// @public
+export function keyframes(timeline: { [key: string]: {} }): string;
+
+// @public
+export function mergeStyles(...args: (IStyle | IStyle[])[]): string;
+
+// @public
+export function mergeStyleSets < T extends {} >(...cssSets: T[]): {[P in keyof T]?: string };

```

Comment:
I think you can use `object` instead of `{}` where you expect actual objects now.